src/helpers/parseString.ts

```diff
@@ -0,0 +1,81 @@
+import { PlayerGender, PlayerParts, HypnoMode } from '../features/gameboard/types'
+import { ISettingsState } from '../features/settings/store'
+
+export function formatMessage(msg: string, settings: ISettingsState) {
+  return msg
+    .replace('$player', transform('player', settings))
+    .replace('$Player', transform('Player', settings))
+    .replace('$part', transform('part', settings))
+    .replace('$Part', transform('Part', settings))
+    .replace('$hands', transform('hands', settings))
+    .replace('$HANDS', transform('HANDS', settings))
+    .replace('$master', transform('master', settings))
+    .replace('$Master', transform('Master', settings))
+    .replace('$stroke', transform('stroke', settings))
+}
+
+function transform(token: string, settings: ISettingsState) {
```

Comment:
This is really clean, since this takes in the settings state obj this will make more dynamic text easier in the future!